src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -0,0 +1,347 @@
+﻿/* Copyright (c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// [Draft] This class provides a View that can layouting items in list and grid with high performance.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class ItemsView : ScrollableBase, ICollectionChangedNotifier
+    {
+        /// <summary>
+        /// Base Constructor
+        /// </summary>
+       [EditorBrowsable(EditorBrowsableState.Never)]
+        public ItemsView() : base()
+        {
+            Scrolling += OnScrolling;
+        }
+
+        /// <summary>
+        /// Base Constructor with itemsSource
+        /// </summary>
+        /// <param name="itemsSource">item's data source</param>
+       [EditorBrowsable(EditorBrowsableState.Never)]
+        public ItemsView(IEnumerable itemsSource) : base()
```

Comment:
I think it may be `: this()`.
```suggestion
        public ItemsView(IEnumerable itemsSource) : this()
```